app/src/main/java/com/bignerdranch/android/androidacademy/MovieViewHolder.kt

```diff
@@ -0,0 +1,45 @@
+package com.bignerdranch.android.androidacademy
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class  MovieViewHolder(
+        view: View,
+        private val listener: OnItemClickListener
+        ): RecyclerView.ViewHolder(view) {
+
+    private val img: ImageView? = view.findViewById(R.id.iv_movie)
```

Comment:
why you need nullable views 
when view.findViewById(R.id.iv_movie) return you not nullable views ?

Reply:
by mistake